docs/integrations/ml-dist.rst

```diff
@@ -0,0 +1,512 @@
+======================================
+Distributed Deep-Learning with CrateDB
+======================================
+
+Abstract
+========
+Using Deep-Learning algorithms for Machine Learning Use-Cases becomes
+more and more common in the world of data science. A common library used
+for solving Deep-Learning problems is `TensorFlow`_, which will be used to
+predict the status of a water pump. CrateDB can easily be integrated into
+an ML-Application as a data source, as well as a data store for the resulting
+predictions. The connection to CrateDB can be established using the `Psycopg2`_
+and `Pandas`_ library. To prepare the data by for example labeling it, we will
+be using the library `Sklearn`_.
+
+Implementation
+==============
+
+Set Up
+------
+For this distributed ML implementation, we will be using a `Kaggle`_ data set
+called `pump_sensor_data`_.
+
+The data set contains data gathered from 52 sensors, with not much other detail
+to them. These data rows come with a timestamp and a machine-status attached to
+them. There are 3 unique states the machine can have, *normal*, *broken*,
+*recovering*. The data is stored in the CrateDB table ``ml.pump_sensors``.
+
+Using this sensor data we want to predict if the machine will fail in a
+specified time window in the future.
+
+Using this data with TensorFlow, we want to accomplish the following:
+
+1. Retrieve data from CrateDB.
+2. Show extracts of how the data looks.
+3. Prepare the data and build a machine learning model that will predict
+   if the machine will fail in a specified time window in the future.
+4. Save the previously built model on AWS S3.
+5. Retrieve the model from AWS S3, predict sample data and insert the results
+   into the CrateDB table ``ml.pump_sensors_predictions``.
+
+Prerequisites
+-------------
+
+- `A running CrateDB cluster`_
+- `A AWS S3 storage bucket`_
+- A AWS config file at ``~/.aws/config`` containing:
```

Comment:
Could you explain why this is needed? I assume for the boto3 library, but it would be worthwhile to say this here.